book/hardware-battery.md

```diff
@@ -1,38 +1,27 @@
 # Battery Interface
```

Comment:
@dmoroni Battery Interface not very clear description of what is contained. Propose we change this to "Battery Charger Interface"